website/docs/r/network_routed_v2.html.markdown

```diff
@@ -129,14 +134,13 @@ Static IP Pools support the following attributes:
 configuration. [More information.][docs-import]
 
 An existing routed network can be [imported][docs-import] into this resource via supplying its path.
-The path for this resource is made of orgName.vdcName.networkName.
+The path for this resource is made of `OrgName.vdcOrVdcGroupName.NetworkName`.
```

Comment:
To be inline with other paths in import sections, maybe use this format:

```suggestion
The path for this resource is made of `org-name.vdc-or-vdc-group-name.network-name`.
```

Reply:
Tuned up here and as well for `imported` and `isolated_v2` networks.